schema/v1-example-statement.json

```diff
@@ -0,0 +1,36 @@
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "subject": [
+    {
+      "name": "oak_functions_loader",
+      "digest": {
+        "sha256": "d4d5899a3868fbb6ae1856c3e55a32ce35913de3956d1973caccd37bd0174fa2"
+      }
+    }
+  ],
+  "predicateType": "https://slsa.dev/provenance/v0.2",
+  "predicate": {
+    "builder": {
+      "id": "https://github.com/project-oak/transparent-release"
+    },
+    "buildType": "https://github.com/project-oak/transparent-release/amber-slsa-buildtype/v1.json",
+    "buildConfig": {
+      "command": "./scripts/runner build-functions-server",
+      "outputPath": "./oak_functions/loader/bin/oak_functions_loader"
+    },
+    "materials": [
+      {
+        "uri": "gcr.io/oak-ci/oak@sha256:53ca44b5889e2265c3ae9e542d7097b7de12ea4c6a33785da8478c7333b9a320",
+        "digest": {
+          "sha256": "15dc16c42a4ac9ed77f337a4a3065a63e444c29c18c8cf69d6a6b4ae678dca5c"
```

Comment:
Why is this hash different from the one in the digest of the URI? Maybe it's obvious to you or others who know more about Docker, but could you add a comment anyways?

Reply:
Nah, the difference in hash is literally just a mistake a made when assembling the example. 😅 Thx for catching, will fix on Mon